### ISSUE TITLE
Bundle torch-cublas-hgemm

### ISSUE THREAD

**Issue by woct0rdho** [Comfy-Org/comfy-kitchen#22]
As you're already using cublas, would you consider bundling [torch-cublas-hgemm](https://github.com/aredden/torch-cublas-hgemm) in this repo? It's been in ComfyUI's `--fast` for a while. Bundling it would make it easier for more people to install it.

**Comment by contentis** [comment]
Can you explain what the added gain would be? PyTorch is already relying on cublasLt for all GEMM operations and it should do the exact same calls. ReLU, GeLU should be possible to be enforced but the seed-up not very large.

If I'm not mistaken you can enable the half accum in PyTorch directly by using https://docs.pytorch.org/docs/stable/backends.html#torch.backends.cuda.matmul.allow_fp16_reduced_precision_reduction 
And I belive this is even a start-up flag in ComfyUI

**Comment by woct0rdho** [comment]
In my impression `--fast cublas_ops` indeed gives speedup when running SDXL, even when `--fast fp16_accumulation` is already enabled. But I may need to do benchmarking again when I have time.

**Comment by woct0rdho** [comment]
Nevermind. After doing benchmarks again and looking at the source code, I can confirm that nowadays `torch.nn.Linear` is as fast as `CublasLinear` when fp16 accum is enabled.
